cloudfront.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  cf_origin_id = "host"
+}
+
+resource "aws_cloudfront_distribution" "geofencing" {
+  provider = aws.iam-Perimeter
+
+  
+  origin {
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols = [
+      "TLSv1.2"]
+    }
+
+    domain_name = "https://startup-sample-project.tnfhhm-dev.nimbus.cloud.gov.bc.ca/"
+    origin_id   = "https://startup-sample-project.tnfhhm-dev.nimbus.cloud.gov.bc.ca/"
+
+  }
+
+  enabled         = true
+  is_ipv6_enabled = true
+  comment         = "geofencing"
+
+  //	- logging should probably be in a central location (centralized-logging account?) - in an aggregated/shared bucket and perhaps also synced into a bucket within the account where the aws-login app is deployed
+  //	- prefix should follow SEA convention like <account>/<region>/<service name> eg. 12345678/ca-central-1/cloudfront
+  //
+  //  logging_config {
+  //    include_cookies = false
+  //    bucket          = "<mylogs>.s3.amazonaws.com"
+  //    prefix          = "geofencing"
+  //  }
+
+  default_cache_behavior {
+    allowed_methods = [
+      "DELETE",
+      "GET",
+      "HEAD",
+      "OPTIONS",
+      "PATCH",
+      "POST",
+    "PUT"]
+    cached_methods = [
+      "GET",
```

Comment:
I'm not sure we'll want to cache all GET requests since the page is dynamic. We won't see updates to the greetings. We could cache static assets like images, css, and js though.

Reply:
Not exactly sure if we can be able to do that as cloudfront allowed methods are ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"] , Or do you want to remove the GET method entirely ?

Reply:
let's remove all caching for now since the main purpose is geofencing.